sdk/python/pulumi_kubernetes/admissionregistration/v1/MutatingWebhookConfiguration.py

```diff
@@ -58,57 +44,47 @@ def __init__(self, resource_name, opts=None, metadata=None, webhooks=None, __nam
         if __opts__ is not None:
             warnings.warn("explicit use of __opts__ is deprecated, use 'opts' instead", DeprecationWarning)
             opts = __opts__
-        if not resource_name:
-            raise TypeError('Missing resource name argument (for URN creation)')
-        if not isinstance(resource_name, str):
-            raise TypeError('Expected resource name to be a string')
```

Comment:
Not sure if these checks were important, or if this is caught by typing improvements now?

Reply:
Looking into this.

Reply:
Yeah this seems covered by typing improvements. Missing name caught by a `TypeError: __init__() missing 1 required positional argument: 'resource_name'`, wrong type caught by `TypeError: Expected resource name to be a string`